### ISSUE TITLE
"Cannot process activity Vie" appears in the log.

### ISSUE THREAD

**Issue by PYU224** [Chocobozzz/PeerTube#5963]
### Describe the current behavior

This kind of warn appears, what can be the cause?
```
warn[2023/9/5 2:10:54] Cannot process activity Like.
{
  "err": {
    "stack": "Error: Cannot fetch remote video with url: https://misskey.kindworld.one/notes/9j8bw8nkx0\n    at /var/www/peertube/versions/peertube-v5.2.1/dist/server/lib/activitypub/videos/get.js:27:19\n    at Generator.next (<anonymous>)\n    at fulfilled (/var/www/peertube/versions/peertube-v5.2.1/node_modules/tslib/tslib.js:164:62)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)",
    "message": "Cannot fetch remote video with url: https://misskey.kindworld.one/notes/9j8bw8nkx0"
  }
}
warn[2023/9/5 2:21:49] Cannot process activity View.
{
  "err": {
    "stack": "HTTPError: Response code 401 (Unauthorized)\n    at Request.<anonymous> (/var/www/peertube/versions/peertube-v5.2.1/node_modules/got/dist/source/as-promise/index.js:118:42)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)",
    "message": "Response code 401 (Unauthorized)",
    "name": "HTTPError",
    "responseBody": {
      "type": "https://docs.joinpeertube.org/api/rest-reference.html#section/Errors/unauthorized_request",
      "detail": "Token is invalid",
      "status": 401,
      "code": "unauthorized_request",
      "error": "Token is invalid"
    },
    "responseHeaders": {
      "server": "nginx/1.18.0 (Ubuntu)",
      "date": "Mon, 04 Sep 2023 17:21:49 GMT",
      "content-type": "application/problem+json; charset=utf-8",
      "content-length": "198",
      "connection": "keep-alive",
      "x-powered-by": "PeerTube",
      "x-frame-options": "DENY",
      "tk": "N",
      "access-control-allow-origin": "*",
      "etag": "W/\"c6-Cxn7QIK0foXTXePvUkk5xIS576s\"",
      "cache-control": "max-age=1"
    },
    "statusCode": 401
  }
}
```

### Steps to reproduce

1.
2.
3.

### Additional information

* PeerTube instance:
  * URL:
  * Version:
  * NodeJS version:
  * Ffmpeg version:

* Browser name, version and platforms on which you could reproduce the bug:
* Link to browser console log if relevant:
* Link to server log if relevant (`journalctl` or `/var/www/peertube/storage/logs/`):

**Comment by PYU224** [comment]
PeerTube instance:

URL: arcadia.video
Version: 5.2.1
NodeJS version: 20.5.1
Ffmpeg version: latest

**Comment by PYU224** [comment]
Is it possible to prevent error messages from being displayed?

**Comment by Chocobozzz** [comment]
Hi,

Yes it should be fixed by https://github.com/Chocobozzz/PeerTube/commit/0cc1a698813e50d86b5a2a8400e1e8a9b82461da (will be included in next peertube version, 6.0)
